pandas/io/json/_normalize.py

```diff
@@ -26,12 +26,43 @@ def convert_to_line_delimits(s):
     return convert_json_to_lines(s)
 
 
+def _parse_use_keys(use_keys: Union[str, List, Callable, None]) -> Callable:
+    """
+    Converts different types of use_keys into a callable
+
+    Parameters
+    ----------
+    use_keys : Str, List or Callable, optional, default: None
+        Returns true or false depending on whether to include or exclude a key
+
+        .. versionadded:: 1.0.0
+
+    Returns
+    -------
+    use_key - Callable
+     It Decides on whether to include a key in processing.
+
+    """
+    if callable(use_keys):
+        return use_keys
+
+    if isinstance(use_keys, list):
```

Comment:
these can be elif

Reply:
It's not because each list returns the result right there. So `elif` has no effect